perllib/Open311/PopulateServiceList.pm

```diff
@@ -266,6 +266,19 @@ sub _add_meta_to_contact_cobrand_overrides {
         } if $self->_current_service->{service_code} eq 'SLRS';
     } elsif ($self->_current_body->name eq 'Warwickshire County Council') {
         $contact->set_extra_metadata( id_field => 'external_id');
+    } elsif ($self->_current_body->name eq 'Buckinghamshire County Council') {
+        push @$meta, {
+            code => 'road-placement',
+            datatype => 'singlevaluelist',
+            description => 'Is the fly-tip located on',
+            order => 100,
+            required => 'false',
+            variable => 'true',
+            values => [
+                { key => 'road', name => 'The road' },
+                { key => 'off-road', name => 'Off the road/on a verge' },
+            ],
+        } if $self->_current_service->{service_name} eq 'Flytipping';
```

Comment:
This really feels like something we should push into the cobrand modules at some point.

Reply:
Have pushed extra commit that should do this.

Reply:
yay!